modules/core/client/components/Board.js

```diff
@@ -24,6 +25,7 @@ function selectName(names) {
  */
 export default function Board({
   names = 'bokeh',
+  style = null,
   children,
   onDisplayPhoto = () => {},
   onHidePhoto = () => {},
```

Comment:
Can we entirely remove this two props now? As we `$broadcast` directly.

Although, maybe it's worth addressing the question that by using `$broadcast` directly in the component changes existing patterns of only triggering "actions" by passing in `on*` functions as props. Although we are already triggering API calls from components, but as we haven't yet added the architecture for a proper data layer (see https://github.com/Trustroots/trustroots/issues/1416), it's in a hybrid state.

Whichever data layer we move towards, I think it will end up working _something_ like how redux with react hooks would work (see https://react-redux.js.org/api/hooks#usedispatch) - which basically, dispatches an "action/event" from the component - and the `$broadcast()` call would probably be replaced by this.

The logic here to include the `$broadcast` call in the component was on my suggestion, to avoid "prop drilldown" where we have to start passing `onDisplayPhoto`/`onHidePhoto` to _all_ the static pages (now there are many in react), and this function was ultimately connected to the old angular controller methods that we also want to get rid of (ideally the react page-level components do not need to keep the old angular controller).

But in this intermediate state it might seem like this is making the components too entangled in other parts of the app. So let's progress https://github.com/Trustroots/trustroots/issues/1416 !